lib/index.ts

```diff
@@ -0,0 +1,5 @@
+// Copyright 2021 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+export {render} from './src/node-sass/render';
```

Comment:
Should this also export the named types that are defined in `render.ts`?

Reply:
Yes, good catch